src/database/__init__.py

```diff
@@ -5,13 +5,13 @@
 
 
 class DatabaseConfigOption(enum.Enum):
-    firebase_metadata = 1
-    local_storage_only = 2
+    firebase = 1
+    local = 2
 
-#TODO: Update to read from config file, or replace this with singleton/dependency injection (whatever decided by pipeline)
-DATABASE_CONFIG = DatabaseConfigOption.local_storage_only
+def get_database(database_config: DatabaseConfigOption):
+    if database_config is DatabaseConfigOption['firebase']:
```

Comment:
could this be changed to `DatabaseConfigOption.firebase` ? Feels better for enum values vs accessing as a dict because it's probably going to be picked up by a code checker